src/contexts/UiSharedComponentsContextProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import { createContext } from 'react';
+import type { TFunction } from '../hooks/useTranslations';
+export interface UiSharedComponentsContextProviderInterface {
+	history: unknown;
```

Comment:
We could fix this typing and name now to be `navigate: NavigateFunction`, right?

Reply:
yep done in a separate commit